dask/bag/core.py

```diff
@@ -1311,7 +1303,15 @@ def from_imperative(values):
 
 
 def merge_frequencies(seqs):
-    return list(merge_with(sum, map(dict, seqs)).items())
+    first, rest = seqs[0], seqs[1:]
+    if not rest:
+        return first
+    out = defaultdict(int)
+    out.update(first)
```

Comment:
Yeah. No need to iterate through this in python land for the first dictionary, as all fields will be empty. Also allows a quick return if only one dictionary.